### DISCUSSION TITLE
Active support of bots via webhooks and Fediverse accounts.

### DISCUSSION THREAD

**Discussion by linux-lukas** [owncast/owncast#1961] [General Owncast discussion]
Inspired by the "bots" of Twitch, which are based on written commands !help, for example, the idea would be that either bots in the form of Fediverse accounts that would regularly send message(s) to followers and appear in the chat. On the other hand, based on the DIY programmed bots of Achnina, who streams on Twitch, also support that bots can be programmed / created that listen to webhooks.

**Comment by gabek** [comment]
I'm not sure if it's what you're looking for, but there is an API for sending out messages to followers from Owncast.

https://owncast.online/api/latest/#tag/Admin/paths/~1api~1admin~1federation~1send/post

**Reply by linux-lukas** [reply]
Could $user filter for COmmand messages such as !help via this API and then use this as a trigger for a message in the chat? if so, then it would be perfect!

**Comment by gabek** [comment]
It's your code, so you can absolutely do whatever you'd like. There's a couple examples of chat bots people have built here if you'd like to see what others have done:

https://owncast.online/examples/projects/
